apps/src/dropletUtils.js

```diff
@@ -60,7 +60,7 @@ exports.getTime = function () {
 exports.promptNum = function (text) {
   var val;
   do {
-    val = parseInt(window.prompt(text), 10);
+    val = Number(window.prompt(text));
```

Comment:
It might be worth briefly considering `parseFloat`, if you want strings like `"11 "` (trailing space) to be acceptable: http://stackoverflow.com/questions/12227594/which-is-better-numberx-or-parsefloatx however either is really probably fine.